Jib jvm build permissions issues

### Describe the bug

While creating a container image using jib (fast-jar) the whole /work folder (content included) is created with 755 and root as owner even when the username is specified or the image run as another user.
which in my case lead to this issue
`Caused by: com.arjuna.ats.arjuna.exceptions.ObjectStoreException: ARJUNA012225: FileSystemStore::setupStore - cannot access root of object store: ObjectStore/ShadowNoFileLockStore/defaultStore/
`

### Expected behavior

the /work folder and sub directories to be created with the specified (or image default) user as owner


### Actual behavior

the /work folder is created with root as owner

### How to Reproduce?

use jib to build the container image as fast-jar

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

11

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

2.7.1

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

3.8

### Additional information

_No response_

--- a/extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java
+++ b/extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java
@@ -342,7 +342,7 @@ public class JibProcessor {
         Path componentsPath = sourceJarBuildItem.getPath().getParent();
         Path appLibDir = componentsPath.resolve(JarResultBuildStep.LIB).resolve(JarResultBuildStep.MAIN);
 
-        AbsoluteUnixPath workDirInContainer = AbsoluteUnixPath.get("/work");
+        AbsoluteUnixPath workDirInContainer = AbsoluteUnixPath.get("/home/jboss");
 
         List<String> entrypoint;
         if (jibConfig.jvmEntrypoint.isPresent()) {

